### ISSUE TITLE
More Timer Icons

### ISSUE THREAD

**Issue by xCJPECKOVERx** [Templarian/MaterialDesign#5583]
I have:

- [x] [Searched all issues](https://github.com/Templarian/MaterialDesign/issues) to make sure there isn't a request for this icon.
- [x] [Searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't exist.
- [x] Only requested a single icon (or a few near-identical ones) in this issue. 

## Usage

More few timer icons would be useful, representing 5 seconds, 20 seconds, 30 seconds. Specifically I'm looking for icons similar to timer-3 and timer-10, just more time variations.

**Comment by Xenomorph99** [comment]
I agree, it would be useful to have a few more `timer` options following the [timer-10](https://materialdesignicons.com/icon/timer-10) and [timer-3](https://materialdesignicons.com/icon/timer-3) pattern. Our `fast-forward` and `rewind` icons have a pattern of 5, 10, 30, and 60 seconds. It would probably be good to have a similar set:

[fast-forward-5](https://materialdesignicons.com/icon/fast-forward-5)
[fast-forward-10](https://materialdesignicons.com/icon/fast-forward-10)
[fast-forward-30](https://materialdesignicons.com/icon/fast-forward-30)
[fast-forward-60](https://materialdesignicons.com/icon/fast-forward-60)

[rewind-5](https://materialdesignicons.com/icon/rewind-5)
[rewind-10](https://materialdesignicons.com/icon/rewind-10)
[rewind-30](https://materialdesignicons.com/icon/rewind-30)
[rewind-60](https://materialdesignicons.com/icon/rewind-60)

**Comment by mathias-berg** [comment]
I would also like a `rewind-15` and `fast-forward-15`!

**Comment by goyney** [comment]
I've added the 15 variation for rewind and fast forward:
[![fast-forward-15](https://materialdesignicons.com/icon/fast-forward-15)](https://materialdesignicons.com/icon/fast-forward-15)[![rewind-15](https://materialdesignicons.com/icon/rewind-15)](https://materialdesignicons.com/icon/rewind-15)

**Comment by goyney** [comment]
We are not going to be able to add any more `timer` icons unless we change the design of them. There is not enough room; Google has the numbers going from the first point to the last. We might consider redoing these in our font, but we'd need to figure out the lower case s. Not a huge problem, but would need to be done.
